setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="Openfisca-Paris",
-    version="5.5.11",
+    version="5.6.0",
```

Comment:
Il n'y a pas d'ajout de nouvelle fonctionnalité, d'après [semver](https://semver.org/lang/fr/) il faudrait plutôt monter le 3e numéro (correctif) 

```suggestion
    version="5.5.12",
```